src/views/Map/Map.tsx

```diff
@@ -15,15 +15,18 @@ import { Copyrights } from '~components/Copyrights/Copyrights';
 import { shortToolbar, toolbar } from '~features/toolbar';
 import { panelClasses } from '~components/Panel';
 import { ToolbarPanel } from '~features/toolbar/components/ToolbarPanel/ToolbarPanel';
-import s from './Map.module.css';
 import { Layout } from './Layouts/Layout';
+import s from './Map.module.css';
 
-const { EditFeaturesOrLayerPanel } = lazily(
-  () =>
-    import(
-      '~features/create_layer/components/EditFeaturesOrLayerPanel/EditFeaturesOrLayerPanel'
-    ),
-);
+const EditPanel = () => {
+  const { EditFeaturesOrLayerPanel } = lazily(
+    () =>
+      import(
+        '~features/create_layer/components/EditFeaturesOrLayerPanel/EditFeaturesOrLayerPanel'
+      ),
+  );
+  return <EditFeaturesOrLayerPanel />;
+};
```

Comment:
Much more simpler way to do it
```
const { EditFeaturesOrLayerPanel: EditPanel } = lazily(
  () =>
    import(
      '~features/create_layer/components/EditFeaturesOrLayerPanel/EditFeaturesOrLayerPanel'
    ),
);
```

Reply:
that way it will always include dependency and toolbar control will be created in included code regardless of feature flag